src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke.exception;
+
+public class DukeException extends Exception {
```

Comment:
That's true, I didn't realise until you brought it up!

I think the additional exceptions that extend Exception are still useful in the sense that you can tell what kind of exception they are describing, rather than just calling DukeException("error text here") which is rather generic.

Reply:
I think the exceptions thrown should be as specific as possible